keyboards/handwired/3dp660/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+LTO_ENABLE = YES
```

Comment:
What you could do here is set `LTO_ENABLE = yes` at keyboard level, and delete this file and remove LTO from the VIA keymap's rules.mk.

Reply:
I just followed along with merlin on a YT video, so I'm a little green on what you're talking about. What do you mean by "at keyboard level"? Where would I place the LTO line? In the rules.mk in main 3dp660 folder, not keymaps folder? Or somewhere else?

Reply:
Basically, yes. Anything you set in the keyboard-level rules.mk is inherited by the keymap, unless you override it in the keymap's rules.mk. So you can put the LTO_ENABLE in the keyboard rules.mk, and it will be applied for both (all) keymaps.

Reply:
Understood, thank you. I'll do that.